package.yaml

```
@@ -0,0 +1,2 @@
+name: watchdog
+description: Improved watchdog support.
```

It isn't super clear what this is an improvement of. Maybe something like convenient, easy-to-use, rich, high-level would be better. Alternatively, it could be "Watchdog support for containers." or some such.

```suggestion
description: Improved watchdog support.
```

---

done.